subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -132,22 +132,23 @@ endif # BT_TBS
 
 ##################### Call Control Client #####################
 
+config BT_TBS_CLIENT_GTBS
+	bool "Generic Telephone Bearer Service client support"
+	help
+	  This option enables support for the GTBS-oriented Call Control client.
+
+config BT_TBS_CLIENT_TBS
+	bool "Telephone Bearer Service client support"
+	help
+	  This option enables support for the TBS-oriented Call Control client.
```

Comment:
We have an issue where we want to make GTBS mandatory and TBS optional: https://github.com/zephyrproject-rtos/zephyr/issues/42487

Is there any way this can accommodate that requirement?

Reply:
Perhaps we should just have `BT_TBS_CLIENT_TBS` which initializes the GTBS (and remove `BT_TBS_CLIENT_GTBS`), and  make `BT_TBS_CLIENT_MAX_TBS_INSTANCES` default to 0?